verification-annotations/src/klee.rs

```diff
@@ -22,12 +22,68 @@ extern "C" {
     fn klee_abort() -> !;
     fn klee_silent_exit(_ignored: u32) -> !;
     fn klee_is_replay() -> i32;
+    fn klee_open_merge();
+    fn klee_close_merge();
+    fn klee_get_value_i32(x: i32) -> i32;
+    fn klee_get_value_i64(x: i64) -> i64;
+    fn klee_get_value_f(x: f32) -> f32;
+    fn klee_get_value_d(x: f64) -> f64;
+    fn klee_is_symbolic(x: usize) -> i32;
+}
+
+unsafe fn get_value_i8(x: i8) -> i8 {
+    klee_get_value_i32(x as i32) as i8
+}
+
+unsafe fn get_value_u8(x: u8) -> u8 {
+    klee_get_value_i32(x as i32) as u8
+}
+
+unsafe fn get_value_i16(x: i16) -> i16 {
+    klee_get_value_i32(x as i32) as i16
+}
+
+unsafe fn get_value_u16(x: u16) -> u16 {
+    klee_get_value_i32(x as i32) as u16
+}
+
+unsafe fn get_value_u32(x: u32) -> u32 {
+    klee_get_value_i32(x as i32) as u32
+}
+
+unsafe fn get_value_u64(x: u64) -> u64 {
+    klee_get_value_i64(x as i64) as u64
+}
+
+unsafe fn get_value_i128(x: i128) -> i128 {
+    let x = x as u128;
```

Comment:
This looks like a typo, you probably meant `get_value_u128(x as u128) as i128`

Reply:
I wasn't sure whether I should call get_value_u128 as you suggest or cut and paste the definition and add appropriate casts.
In the end, I decided to go with cut and paste so that it is more standalone.

But I think you're right - the shorter version is easier to read and there is no obvious down-side.
So I'll make the change - thanks.